frontend/src/metabase/models/containers/ModelDetailPage/ModelDetailPage.tsx

```diff
@@ -121,7 +121,13 @@ function ModelDetailPage({
   }, [location.pathname]);
 
   useMount(() => {
-    loadMetadataForCard(model.card());
+    const card = model.card();
+    const isModel = model.isDataset();
+    if (isModel) {
+      loadMetadataForCard(card);
```

Comment:
Should we do this for `/model/:model-id` path too for consistency?

Now we're getting something like this, or do you think this is fine?
<img width="890" alt="image" src="https://user-images.githubusercontent.com/1937582/220179181-2f4e4069-0b60-4f92-9694-73c93c32d464.png">

Reply:
Makes sense to me. Let's bring this to product and issue another PR if they're in 👍